File: Zahlenratespiel.py
import random

for x in range (1):# Für x soll im Bereich (1) :
 zufaeligeZahl = random.randint(1, 100)# Zahlen zwischen (1 und 100) ausgegeben werden.
print("Das ist das Zahlenratespiel. Dein Ziel ist es die zufällige Zahl zu finden")
eingabe = int(input("Gib eine Zahl ein"))

while eingabe != zufaeligeZahl:
 if eingabe > zufaeligeZahl:
  print("Dein Tipp ist größer als die gesuchte Zahl")
  eingabe = int(input("gib bitte einen neuen Tipp ab "))
  #Ende of if-condition

 else:
  print("Dein Tipp ist kleiner als die gesuchte Zahl")
  eingabe = int(input("gib bitte einen neuen Tipp ab "))
  #End of else-condition
  
#still part of while-loop 
print("Glückwunsch! Du hast die richtige Zahl erraten. ")
print("Die gesuchte Zahl war") 
print(zufaeligeZahl)
#End of program

File: Zahlenratespiel.java
import javax.swing.JOptionPane;


public class Zahlenratespiel {
	
	public static void main(String[] args) {
		float generierteZahl = (float) Math.random() * 100;
		int zufaeligeZahl = (int) generierteZahl;
		String eingabe = JOptionPane.showInputDialog("Gib eine Zahl ein");
		int tipp = Integer.parseInt(eingabe);
		
		while(tipp != zufaeligeZahl) {
			if(tipp < zufaeligeZahl) {
				JOptionPane.showMessageDialog(null, "Deine Zahl ist kleiner als die gesuchte Zahl");
				eingabe = JOptionPane.showInputDialog("Gib eine neuen Tipp ab!");
				tipp = Integer.parseInt(eingabe);
			} else if(tipp > zufaeligeZahl) {
				JOptionPane.showMessageDialog(null, "Deine Zahl ist größer als die gesuchte Zahl");
				eingabe = JOptionPane.showInputDialog("Gib eine neuen Tipp ab!");
				tipp = Integer.parseInt(eingabe);
				
			}
		}
		
		JOptionPane.showMessageDialog(null, "Dein Tipp war richtig! Die gesuchte Zahl war " + zufaeligeZahl);
	}
}
